css-ecommerce-page/index.html

```diff
@@ -0,0 +1,90 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Ecommerce Challenge</title>
+  <link href="https://fonts.googleapis.com/css?family=DM+Sans&display=swap" rel="stylesheet">
+  <link href="styles.css" rel="stylesheet">
+</head>
+<body>
+  <div class="container">
+    <div class="row header-arrivals">
+      <h1 class=""> New Arrivals </h1>
+    </div>
+    <div class="row">
+      <div class="column-full">
```

Comment:
This column takes up only one fifth of the width, right? Why is it set as `column-full`? The rest of your columns are the same way, you should revise this to correctly describe its content.